# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libquirc/hotfix-makefile.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/Makefile
+++ b/Makefile
@@ -15,8 +15,9 @@
 
 CC ?= gcc
 PREFIX ?= /usr/local
-SDL_CFLAGS ?= pkg-config --cflags sdl 
-SDL_LIBS ?= pkg-config --libs sdl
+LIBDIR ?= $(PREFIX)/lib
+SDL_CFLAGS ?= $(pkg-config --cflags sdl)
+SDL_LIBS ?= $(pkg-config --libs sdl)
 
 LIB_VERSION = 1.0
 
@@ -62,12 +63,11 @@ libquirc.so.$(LIB_VERSION): $(LIB_OBJ)
 .c.o:
 	$(CC) $(QUIRC_CFLAGS) -o $@ -c $<
 
-install: libquirc.a libquirc.so.$(LIB_VERSION) quirc-demo quirc-scanner
+install: libquirc.a libquirc.so.$(LIB_VERSION) quirc-scanner
 	install -o root -g root -m 0644 lib/quirc.h $(DESTDIR)$(PREFIX)/include
-	install -o root -g root -m 0644 libquirc.a $(DESTDIR)$(PREFIX)/lib
+	install -o root -g root -m 0644 libquirc.a $(DESTDIR)$(LIBDIR)
 	install -o root -g root -m 0755 libquirc.so.$(LIB_VERSION) \
-		$(DESTDIR)$(PREFIX)/lib
-	install -o root -g root -m 0755 quirc-demo $(DESTDIR)$(PREFIX)/bin
+		$(DESTDIR)$(LIBDIR)
 	install -o root -g root -m 0755 quirc-scanner $(DESTDIR)$(PREFIX)/bin
 
 uninstall:
